lib/templates/bootstrap/textbox.js

```diff
@@ -69,7 +72,7 @@ function textbox(locals) {
           onChange={locals.onChangeNative}
           placeholder={locals.placeholder}
           style={textboxStyle}
-          value={locals.value}
+          value={locals.value ? locals.value.toString() : ''}
```

Comment:
Will this set empty textboxes to an empty string? I think `undefined` would be preferred

Reply:
I'll check this. I was getting an error saying the field had to be a string, but that was before I tracked down another issue so it may be resolved without this.